tests/test_cvi.py

```diff
@@ -11,41 +11,206 @@
 import os
 from pathlib import Path
 import logging as lg
+# import inspect
 
 # --------------------------------------------------------------------------- #
 # CUSTOM IMPORTS
 # --------------------------------------------------------------------------- #
 
 import pytest
 import numpy as np
+import pandas as pd
 
 # --------------------------------------------------------------------------- #
 # LOCAL IMPORTS
 # --------------------------------------------------------------------------- #
+
 print(f"\nTesting path is: {os.getcwd()}")
 import src.cvi as cvi
 # import ..
 
+# --------------------------------------------------------------------------- #
+# FIXTURES
+# --------------------------------------------------------------------------- #
+
+# Set the fixture scope to the testing session to load the data once
+@pytest.fixture(scope="session")
+def data() -> dict:
```

Comment:
use "from Typing import Dict" and the define the return type with -> Dict[str,Dict[str,Dict[str,np.ndarray]]]

Reply:
While that would be more specific, I think that I might just make a new TestData container to standardize things. I'll update the signature typing accordingly.